src/Workspace.scss

```diff
@@ -12,6 +12,10 @@ $border-light: 1px solid $light-color;
 
 $font: "Nunito", sans-serif;
 
+$ph-gutter: 10;
+$ph-border: 0;
+@import "../node_modules/placeholder-loading/src/scss/placeholder-loading.scss";
```

Comment:
i wonder that there is no error/warning, but please move to line 3 to other imports for overview reasons

Reply:
unfortunately I need to set the $ph-gutter and $ph-border variable before the import